lnrpc/wtclientrpc/wtclient.proto

```diff
@@ -0,0 +1,137 @@
+syntax = "proto3";
+
+package wtclientrpc;
+
+option go_package = "github.com/lightningnetwork/lnd/lnrpc/wtclientrpc";
+
+message AddTowerRequest {
+    // The identifying public key of the watchtower to add.
+    bytes pub_key = 1 [json_name = "pub_key"];
+
+    // A network address the watchtower is reachable over.
+    string address = 2 [json_name = "address"];
+}
+
+message AddTowerResponse {
+}
+
+message RemoveTowerRequest {
+    // The identifying public key of the watchtower to remove.
+    bytes pub_key = 1 [json_name = "pub_key"];
+
+    /*
+    If set, then the record for this address will be removed, indicating that is
+    is stale. Otherwise, the watchtower will no longer be used for future
+    session negotiations and backups.
+    */
+    string address = 2 [json_name = "address"];
+}
+
+message RemoveTowerResponse {
+}
+
+message GetTowerInfoRequest {
+    // The identifying public key of the watchtower to retrieve information for.
+    bytes pub_key = 1 [json_name = "pub_key"];
+}
+
+message TowerSession {
+    /*
+    The total number of successful backups that have been made to the
+    watchtower.
+    */
+    uint32 num_backups = 1 [json_name = "num_backups"];
+
+    /*
+    The total number of backups that are currently pending to be acknowledged by
+    the watchtower.
+    */
+    uint32 num_pending_backups = 2 [json_name = "num_pending_backups"];
+
+    // The total number of backups allowed by the watchtower.
+    uint32 num_max_backups = 3 [json_name = "num_max_backups"];
+
+    /*
+    The fee rate, in satoshis per vbyte, that will be used by the watchtower for
+    the justice transaction in the event of a channel breach.
+    */
+    uint32 sweep_sat_per_byte = 4 [json_name = "sweep_sat_per_byte"];
+}
+
+message Tower {
+    // The identifying public key of the watchtower.
+    bytes pub_key = 1 [json_name = "pub_key"];
+
+    // The list of addresses the watchtower is reachable over.
+    repeated string addresses = 2 [json_name = "addresses"];
+
+    // Whether the watchtower is currently a candidate for new sessions.
+    bool active_session_candidate = 3 [json_name = "active_session_candidate"];
+
+    // The list of sessions currently active with the watchtower.
+    repeated TowerSession sessions = 4 [json_name = "sessions"];
```

Comment:
also add `num_sessions` in case `sessions` isn't populated?

Reply:
Done.